contracts/axelar-gas-service/src/error.rs

```diff
@@ -0,0 +1,12 @@
+use soroban_sdk::contracterror;
+
+#[contracterror]
+#[derive(Copy, Clone, Debug, Eq, PartialEq, PartialOrd, Ord)]
+#[repr(u32)]
+pub enum ContractError {
+    InvalidAddress = 1,
```

Comment:
Just wondering, how are we going to handle the order of error names? Should they be in alphabetical order or based on occurence? In other contracts, NotInitialized / AlreadyInitialized are assigned higher numbers. (e.g. [GatewayError::NotInitialized](https://github.com/axelarnetwork/axelar-cgp-soroban/blob/3e7d28a8806fec2c689989b2e50de1860587190c/contracts/axelar-soroban-interfaces/src/axelar_gateway.rs#L12) comes first). I think it would be great to have some kind of convention or simple rules for this.

Reply:
you can see if Rust recommends something. We should avoid specifying error enum values though, and let it use the defaults

Reply:
i don't think there's a particular recommendation. it'd be nice if they were all aligned but not really that important since (as far as i know) you rarely interact with the values behind integer variants explicitly.